### PR TITLE
Fix compilation with chunked feature

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
The trussed-staging dependency is out of date

This puts `trussed` back to `Nitrokey/trussed` main and staging to the `hmacsha256p256` branch rebased on main.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -62,8 +62,8 @@ ctaphid-dispatch = { git = "https://github.com/trussed-dev/ctaphid-dispatch.git"
 apdu-dispatch = { git = "https://github.com/trussed-dev/apdu-dispatch.git", rev = "915fc237103fcecc29d0f0b73391f19abf6576de" }
 littlefs2 = { git = "https://github.com/trussed-dev/littlefs2.git", rev = "ebd27e49ca321089d01d8c9b169c4aeb58ceeeca" }
 serde-indexed = { git = "https://github.com/sosthene-nitrokey/serde-indexed.git", rev = "5005d23cb4ee8622e62188ea0f9466146f851f0d" }
-trussed = { git = "https://github.com/trussed-dev/trussed.git", rev = "cff2e663841b6a68d3a8ce12647d57b2b6fbc36c" }
+trussed = { git = "https://github.com/Nitrokey/trussed.git", rev = "cb7bd328549293077cd9d6c2e9e42df62180db96" }
```
Comment (robin-nitrokey) [inline Cargo.toml:65]: Please use the tag [v0.1.0-nitrokey.18](https://github.com/Nitrokey/trussed/releases/tag/v0.1.0-nitrokey.18).

File: Cargo.toml [hunk 2]
```diff
@@ -62,8 +62,8 @@ ctaphid-dispatch = { git = "https://github.com/trussed-dev/ctaphid-dispatch.git"
...
 trussed-hkdf = { git = "https://github.com/Nitrokey/trussed-hkdf-backend.git", tag = "v0.1.0" }
-trussed-staging = { git = "https://github.com/trussed-dev/trussed-staging", rev = "3b9594d93f89a5e760fe78fa5a96f125dfdcd470" }
+trussed-staging = { git = "https://github.com/Nitrokey/trussed-staging", rev = "fc90ab461bb288e3944dd9c80e2611590b2cfa22" }
```
Comment (robin-nitrokey) [inline Cargo.toml:67]: We really need to get these branches merged ... Can we use [v0.1.0-nitrokey-hmac256p256.3](https://github.com/Nitrokey/trussed-staging/releases/tag/v0.1.0-nitrokey-hmac256p256.3) here?

File: Cargo.toml (hunk 1 above)
Comment (sosthene-nitrokey) [reply Cargo.toml]: Done

File: Cargo.toml (hunk 2 above)
Comment (sosthene-nitrokey) [reply Cargo.toml]: I don't think we should merge the hmac-sha into trussed-staging. We could put it in its own repo in another backend + extension though.

**Review by robin-nitrokey — APPROVED** [review]
Thank you!

**Pull request merged**